server/lib/db.ts

```diff
@@ -79,8 +83,8 @@ export default () => {
           db.set(
             "search_data",
             data.slice(1).map(counselor => ({
-              id: counselor[0],
-              text: counselor.join(" "),
+              id: counselor.licId,
+              text: JSON.stringify(counselor),
```

Comment:
What is the purpose of this filed? Do we still need this?

Reply:
So this is how we implement searching currently. We use the text field to simulate full-document searching by matching the user query against the entire counselor object. In the (near) future I want to implement specific field searching so for now we can leave this in, as it will be getting refactored.